tests/browsing_context/test_navigate_events.py

```diff
@@ -180,7 +180,52 @@ async def test_scriptNavigate_aboutBlank_checkEvents(websocket, context_id,
             }
         })
 
-    messages = await read_sorted_messages(9,
+    messages = await read_sorted_messages(7,
+                                          keys_to_stabilize=KEYS_TO_STABILIZE,
+                                          check_no_other_messages=True)
+    assert messages == snapshot(exclude=SNAPSHOT_EXCLUDE)
+
+
+@pytest.mark.asyncio
+async def test_aboutBlank_reload_checkEvents(websocket, context_id, html,
```

Comment:
let's add tests for history updated and fragment navigations in before unload?

Reply:
follow-up?